src/ShopBundle/Resources/views/snippets/ShopBundle/BasketForm/hiddenBasketFields.html.twig

```diff
@@ -0,0 +1,3 @@
+{% for name, value in values %}
+<input type="hidden" name="{{name | e('html_attr')}}" value="{{value | e('html_attr')}}" />
```

Comment:
does that work when you have nested parameter (foo=>['bar'=>'baz']) ?

Reply:
No it does not. And it will crash the replacer if there are such parameters. This is of course not acceptable so I pushed an update that flattens the input before rendering.